### DISCUSSION TITLE
Leave wezTerm running when last window closed on macos

### DISCUSSION THREAD

**Discussion by tammersaleh** [wezterm/wezterm#3290] [Q&A] [answered]
Is there a way to leave wezTerm running when the last window is closed on macos?  I'd like to be able to open a new window via `CMD+n` when no window are open, but because wesTerm terminates when the last window is closed, I have to re-launch wezTerm instead.

I know [this is expected behavior](https://wezfurlong.org/wezterm/config/lua/keyassignment/CloseCurrentTab.html), but I was hoping there was a way of overriding it.

**Comment by wez** [comment, marked as answer]
In nightly builds: https://wezfurlong.org/wezterm/config/lua/config/quit_when_all_windows_are_closed.html
This is supported only on macOS right now.

**Reply by tammersaleh** [reply]
perfect - thank you!

**Reply by cxwx** [reply]
perfect！

**Reply by wvangeit** [reply]
Very nice!

**Reply by AttakDoge** [reply]
Sorry for the necro, but is this still macOS only?

**Reply by wavebeem** [reply]
I feel like this should be true on macOS by default. I came here via search to figure out why the app was closing. It messes with my ability to Cmd-W Cmd-N and makes wezterm feel slower to me because starting the app will just open a new window if the app is already running, vs having to load all the code first.
